portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -240,12 +240,37 @@ definition {
 	}
 
 	@description = "Verify that the Field Title can be edited"
-	@ignore = "true"
 	@priority = "5"
 	test TitleCanBeEdited {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131226 TitleCanBeEdited pending implementation
+	ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+	FormsAdminNavigator.gotoAddForm();
+
+	Form.editName(formName = "Form 1");
+
+	Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+	FormFields.editParagraphBody(content = "Test");
+
+	Form.editFieldLabelMetal(fieldValue = "Test Paragraph");
+
+	Form.save();
+
+	Click(locator1 = "Icon#BACK");
+
+	FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+	AssertElementPresent(
+		locator1 = "FormFields#PARAGRAPH_TITLE",
+		key_fieldName = "Paragraph",
+		value1 = "Test Paragraph");
```

Comment:
@piresdm, how about use a macro created exactly for this same purpose, e.g. the macro `FormFields.viewParagraphField`?

Reply:
Hi @ThalitaCeli, can you please make this change? Thanks!

Reply:
Hello, @brittneyq. Done.
Need to make any more changes?
Thanks.

Reply:
That should be all for now. Thanks @ThalitaCeli !